### DISCUSSION TITLE
smplayer without sound

### DISCUSSION THREAD

**Discussion by satimis** [smplayer-dev/smplayer#951] [General]
Hi all,
smplayer
Version: 23.12.0 (revision 10242)
OS - Ubuntu 22.04 desktop
There is no sound on video CDs.  Other video player such as VLS media player, Video etc. works with sound output.
Please help.  Thanks in advance.
Edit: the volume control works but no sound emitted
Regards

**Comment by smplayer-dev** [comment]
Could you send the log (menu View) after playing a video with this problem?

**Reply by satimis** [reply]
Hi,

Thanks for your advice.
After re-installing smplayer on Ubuntu repo, sound comes back.  Now I can play VCD on smplayer with sound.
Besides, can I convert VCD to MP4 on smplayer?  Thanks
Regards
